internal/openapi/loader/SpecificationLoader.go

```diff
@@ -9,7 +9,7 @@ type SpecificationLoader interface {
 func New() SpecificationLoader {
 	return &processingLoader{
 		&autoLoader{
-			loader: openapi3.NewLoader(),
+			loader: &openapi3.Loader{IsExternalRefsAllowed: true},
```

Comment:
Hi! Thanks for the PR. Perhaps this behavior should be configurable. What do you think?

I suggest adding options.

For package loader.

```golang
type Options struct {
	IsExternalRefsAllowed bool
}

func New(options Options) SpecificationLoader {
	return &processingLoader{
		&autoLoader{
			loader: &openapi3.Loader{
				IsExternalRefsAllowed: options.IsExternalRefsAllowed,
			},
		},
	}
}
```

YAML option

```yaml
# OpenAPI specification options
openapi:
  specification_url: 'path/to/your/openapi-specification.yaml'
  allow_external_refs: true
```

In config package

```golang
type Configuration struct {
	// OpenAPI options
	SpecificationURL  string
	AllowExternalRefs bool
```

Reply:
I agree. The PR in its current form is for a "PoC" if you like. I will add this and update the PR.

@strider2038 What to do about the failing tests in the CI. They pass locally on my machine. Can you confirm it's the same for you?

Reply:
I thought a little more. It seems that it is better to leave the behavior as simple as possible. If it is necessary for security reasons, it is better to add an option to disable reference reading.

> What to do about the failing tests in the CI. They pass locally on my machine. Can you confirm it's the same for you?

It looks like these are some issues with scrutinizer in newer versions of Go.